### PR TITLE
Task1.1 

### PR DESCRIPTION
**Pull request by IRIKA**
Task 1.1 is completed
Unit tests have added

### CODE REVIEW

File: GameStore/GameStore.BLL.Tests/Services.Tests/CommentServiceTests.cs [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+            // arrange
+            _mockGameRepo.Setup(x => x.GetByField(It.IsAny<Func<Game, bool>>(), x => x.Comments, x => x.Genres, x => x.PlatformTypes)).Returns(_dummyGame);
+            _mockMapper.Setup(m => m.Map<GameDTO, Game>(It.IsAny<GameDTO>())).Returns(_dummyGame);
+            _mockMapper.Setup(m => m.Map<Comment>(It.IsAny<CommentDTO>())).Returns(_dummyComment);
+            
+            // act
+            _testedCommentService.AddComment(_dummyGame.Key, _dummyCommentDTO);
+
+            // assert
+            _mockUnitOfWork.Verify(x => x.SaveChanges());
```
Comment (IrynaShemet) [inline GameStore/GameStore.BLL.Tests/Services.Tests/CommentServiceTests.cs:63]: Verify create method from comment repository

File: GameStore/GameStore.BLL.Tests/Services.Tests/GameServiceTests.cs [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+        private List<Game> _dummyListGame;
+        private List<CommentDTO> _dummyListCommentDTO;
+        private List<Comment> _dummyListComment;
+        private List<GenreDTO> _dummyListGenresDTO;
+        private List<Genre> _dummyListGenres;
+        private List<PlatformTypeDTO> _dummyListPlatformTypesDTO;
+        private List<PlatformType> _dummyListPlatformTypes;
+
+        [TestInitialize()]
+        public void Initialize()
```
Comment (IrynaShemet) [inline GameStore/GameStore.BLL.Tests/Services.Tests/GameServiceTests.cs:35]: Add test for delete

File: GameStore/GameStore.BLL.Tests/Services.Tests/GameServiceTests.cs [hunk 3]
```diff
@@ -0,0 +1,285 @@
...
+            // act
+            var result = _testedGameService.GetById(id);
+
+            // assert
+            Assert.AreEqual(expected, result);
+        }
+
+        [TestMethod()]
+        public void GetGameByKeyMethod_AnyString_GameReturned()
+        {
```
Comment (IrynaShemet) [inline GameStore/GameStore.BLL.Tests/Services.Tests/GameServiceTests.cs:97]: Key can be different case

File: GameStore/GameStore.BLL.Tests/Services.Tests/CommentServiceTests.cs (hunk 1 above)
Comment (IRIKA) [reply GameStore/GameStore.BLL.Tests/Services.Tests/CommentServiceTests.cs:63]: Create method from comment repository has been verified

File: GameStore/GameStore.BLL.Tests/Services.Tests/GameServiceTests.cs (hunk 3 above)
Comment (IRIKA) [reply GameStore/GameStore.BLL.Tests/Services.Tests/GameServiceTests.cs:97]: The GetByKey(key) method can take a parameter in any register - uppercase or lowercase

**Pull request merged**